app/javascript/onboarding/components/IntroSlide.jsx

```diff
@@ -22,6 +23,9 @@ class IntroSlide extends Component {
 
   render() {
     const { prev } = this.props;
+    const greeting = this.user
+      ? `${this.user.name} — welcome to DEV!`
```

Comment:
would this always be true? I noticed how in the utilities this parses JSON. Is there a scenario where the server sends null in the dataset?

Reply:
Yes! The only reason I had this was as a fallback when I couldn't figure out why the user wasn't there in tests. I'm going to remove and just use the expected greeting :)